cmake/modules/FindLilv.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Dalton Messmer <messmer.dalton/at/gmail.com>
+#
+# Redistribution and use is allowed according to the terms of the New BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
+
+include(ImportedTargetHelpers)
+
+find_package_config_mode_with_fallback(lilv lilv::lilv
```

Comment:
Curious why this isn't `Lilv` `Lilv::lilv` (and then `PREFIX` can be omitted)

Reply:
Looks like it's broken now:
```
-- Checking for module 'lv2'
--   Found lv2, version 1.18.10
-- Could NOT find Lilv (missing: Lilv_LIBRARY Lilv_INCLUDE_DIRS) 
```

Did I do something wrong?

Reply:
> Did I do something wrong?

I don't know Dom's scripts too well, so my advice may have been wrong.  I'll see what I can learn.

Reply:
Oh, the first `lilv` should be capitalized.